src/main/resources/assets/minecolonies/gui/layouthuts/layouthutpageactionsminwoinv.xml

```diff
@@ -5,7 +5,7 @@
     <image source="minecolonies:textures/gui/builderhut/builder_sketch_left.png" size="6 15" pos="44 12"/>
     <image source="minecolonies:textures/gui/builderhut/builder_sketch_center.png" size="90 15" pos="50 12"/>
     <image source="minecolonies:textures/gui/builderhut/builder_sketch_right.png" size="6 15" pos="140 12"/>
-    <label id="name" size="90 11" pos="50 14" textalign="MIDDLE" color="red"/>
+    <label id="name" size="100 11" pos="46 14" textalign="MIDDLE" color="red"/>
```

Comment:
When you enlarge width by 10 the pos x shift is then -5, so either width = 98 or pos_x = 45

Reply:
Depends on how it looks like in the UI after all, potentially the existing one was unaligned

Reply:
I'll fix it